packages/php/email-editor/README.md

```diff
@@ -57,3 +79,15 @@ Please use with caution.
 | `mailpoet_email_content_renderer_styles`   | `string` $contentStyles, `WP_Post` $post  | `string` $contentStyles                                      | Applied to the inline content styles prior to use by the CSS Inliner.                                                                                               |
 | `mailpoet_is_email_editor_page`            | `boolean` $isEditorPage                   | `boolean`                                                    | Check current page is the email editor page                                                                                                                         |
 | `mailpoet_email_editor_send_preview_email` | `Array` $postData                         | `boolean` Result of processing. Was email sent successfully? | Allows override of the send preview mail function. Folks may choose to use custom implementation                                                                    |
+
+
+## FAQ
+1. How can I activate the new email editor?
+   * You need to active it as an experimental feature on the URL {your-website}/wp-admin/admin.php?page=mailpoet-experimental
+   * After that, you can see a dropdown button when you want to create a new email.
+2. Are emails from the old editor compatible?
+   * No. Unfortunately, email editors are not compatible at this moment, and there is no tool which would allow you to migrate old emails yet.
+3. Which WP versions are supported?
+   * The new email editor supports only the latest WP version. The last supported version was 6.6.
+4. Can I use the latest Gutenberg version?
+   * You can use the latest version by installing the Gutenberg plugin, but there is a high chance that the editor will not work properly.
```

Comment:
This might be a better fit to put to `mailpoet/lib/EmailEditor/README.md` because it is related to the integration in MailPoet.
I would remove `The last supported version was 6.6.` it is already outdated.

Reply:
Noted. Fixed in https://github.com/mailpoet/mailpoet/pull/5959/commits/dd8bc0d1aec393585b4e44e7826854fb93ad9b3b